examples/qclustering/train_qkmedians.py

```diff
@@ -0,0 +1,185 @@
+import argparse
+import os
+
+import h5py
+import numpy as np
+
+import qibo
+
+qibo.set_backend("qibojit")
+
+import qkmedians as qkmed
+
+
+def train_qkmedians(
+    train_size,
+    read_file,
+    seed=None,
+    k=2,
+    tolerance=1.0e-3,
+    min_type="classic",
+    nshots=10000,
+    save_dir=None,
+    verbose=False,
+    nprint=10,
+):
+    """Perform training of quantum k-medians.
+
+    Parameters
+    ----------
+    train_size : int
+        Number of training samples.
+    read_file : str
+        Name of the file where training data is saved.
+    seed : int
+        Seed for data shuffling.
+    k : int
+        Number of classes in quantum k-medians.
+    tolerance : float
+        Tolerance for algorithm convergence.
+    min_type : str
+        Type of minimization for distance calculation, classic or quantum.
+    nshots : int
+        Number of shots for executing quantum circuit.
+    save_dir : str
+        Name of the file for saving results.
+    nprint : int
+        Print loss function value each `nprint` epochs if `verbose` is `True`.
+    verbose : str
+        Print log messages if `True`.
```

Comment:
```suggestion
        Print log messages during the training if `True`.
```

Reply:
Will add the picture. Thanks for recommending it. And thanks for the suggestions.